app.js

```diff
@@ -355,6 +389,11 @@ function renderPage( type, options ) {
   var query     = options.query;
   var debug     = options.debug;
   var cssCookie = options.cssCookie;
+  var isTools   = type === 'tools';
+
+  var isDemoToolAdded = isTools ? _.some( list, function( tool ) {
+    return tool.name === '_DEMO TOOL_';
+  } ) : false;
 
```

Comment:
This logic feels kind of in the wrong place for me...

Maybe `addDemoTool` could keep track of this?